package.json

```diff
@@ -5,8 +5,9 @@
   "description": "Static site generator for the HMRC Design System",
   "scripts": {
     "start": "gulp",
-    "test": "gulp lint && jest",
-    "build": "npm run test && gulp build",
+    "test": "gulp lint && jest --watch",
```

Comment:
Can we switch round the watch task to a different name?  That way:

`npm test` will give a status code result
`npm run test:watch` or `npm run dev` will support developers

I like the idea that `npm test` is a consistent feature across projects, I think that consistency should be available on build servers etc.